home/models.py

```diff
@@ -10,61 +10,57 @@
 from modelcluster.models import ClusterableModel
 
 from wagtail.models import Page
-from wagtail.admin.edit_handlers import FieldPanel
 from wagtail.snippets.models import register_snippet
-from modelcluster.fields import ParentalKey
-from modelcluster.models import ClusterableModel
 from taggit.models import TaggedItemBase
 from taggit.managers import TaggableManager
 
 from home.forms import SignUpPage
-from accounts.models import Link
 
 from .managers import EventQuerySet
 
 
 def sign_up_forms(context):
-    return{
-        'sign_up_forms': SignUpPage.objects.all(),
-        'request': context['request'],
+    return {
+        "sign_up_forms": SignUpPage.objects.all(),
+        "request": context["request"],
     }
 
-class HomePage(Page):
-    content_panels = Page.content_panels + [
 
-    ]
+class HomePage(Page):
+    content_panels = Page.content_panels + []
 
     def get_context(self, request):
         context = super().get_context(request)
-        events = Event.objects.visible()
+        events = Event.objects.visible()  # type: ignore
```

Comment:
If this is silencing some type style rule, I would probably ignore that rule entirely. I can't tell by looking at this what it's covering. The project doesn't need all the linting rules enabled. Some is good to start.

Reply:
Yea, I'll roll this back. VS code is working on better Pylance linting and to quite the errors I added these comments to ignore. 

Thanks for the feedback! I agree.